public/styles/scene.css

```diff
@@ -128,13 +120,9 @@ kbd {
   position: absolute;
   font-size: 30px;
   text-transform: uppercase;
-  color: #ffffff;
+  color: var(--color-linen);
   font-weight: 700;
-  font-family: "Linotte";
-}
-
-#smiley {
```

Comment:
I've been trying to figure this out for the last 20 minutes but can't figure out the root issue of this, but the smiley face in scene.html is gone.

karaokenite.co:

<img width="267" alt="karaokenite co" src="https://user-images.githubusercontent.com/4999022/118377898-21b67d00-b59e-11eb-84ca-cb28921a9595.png">

here:

<img width="177" alt="this_branch" src="https://user-images.githubusercontent.com/4999022/118377904-27ac5e00-b59e-11eb-94d8-e73f952477a1.png">

Reply:
Got it, position inheritance from `a`.  Going to remove the styles we are applying directly to tags too so this shouldn't happen.